packages/components/src/index.ts

```diff
@@ -13,21 +14,32 @@ import {
   spaces,
   fonts,
   shadows,
-  TokensTable
+  TokensTable,
 } from "./tokens";
 
 import "./styles/utilities.scss";
 
-export {
+export const components = {
```

Comment:
Toto mi pride celkom sikovne!

Reply:
Ale asi to nefunguje, lebo dostavam odtial `FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory`. Ked zakomentujem funkcie `getMDXComponents` a `getMDXScope`, tak tento problem oddide. Neviem ci tam bol aj predtym, a kvoli tomu to bolo pomale.

Reply:
Toto nebolo relevantne. FIxnute v #10